components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/OAuthUtil.java

```diff
@@ -446,6 +446,7 @@ public static OAuthConsumerAppDTO buildConsumerAppDTO(OAuthAppDO appDO) {
         dto.setTokenRevocationWithIDPSessionTerminationEnabled(appDO
                 .isTokenRevocationWithIDPSessionTerminationEnabled());
         dto.setTokenBindingValidationEnabled(appDO.isTokenBindingValidationEnabled());
+        dto.setOauthExpiryTime(0);
```

Comment:
can we add a cmt here ? explaining why it is set to 0

Reply:
Fixed with https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1747/commits/bb9fe606108cfd608e96a854491223e263178043